content/blog/2019-06-27-updates-to-the-chi-submission-and-reviewing-guides.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: Updates to the CHI Submission and Reviewing Guides Promote Open Research Practices
+author: Steve Haroz
+date: '2019-08-01'
+slug: updates-to-chi-submission-and-reviewing-guides
+categories: []
+tags: [chi]
+---
+
+Over the past year, members of [Transparent Statistics in HCI](https://transparentstatistics.org/) have worked with people across various disciplines of the HCI community and chairs of the conference (full contributor list below) to update the CHI guides for a successful submission and for reviewing. These new instructions aim to increase the transparency and openness of research, including:
+
+* decisions performed
+* materials created or used
+* data collected
+
+The availability of these facets of research can enable reviewers and readers to better assess HCI research, facilitate industry to build upon and apply HCI research, and help HCI researchers to more efficiently use past research to make new advances.
+
+This post highlights some of the updates, but you can read the full [guide to a successful CHI submission](https://chi2020.acm.org/authors/papers/guide-to-a-successful-submission/) and [guide to reviewing](https://chi2020.acm.org/guide-to-reviewing-papers/) yourself.
+
+
+## Transparency
+
+To ensure that research is clearly and accurately communicated, every facet of CHI research is expected to be presented as transparently as possible. If reviewers have serious questions about what was done or how decisions were made, a paper’s contribution may be called into question. Authors can avoid these concerns by describing all methods and analyses along with any warranted precision and uncertainty.
+
+New or updated text in the reviewing guide:
+
+> Lack of transparency in the way research results are reported can be a ground to doubt the contribution. See the “Transparency” section in “Guide to a Successful Submission” for a discussion of transparency in different contribution types.
```

Comment:
Because there is stylizing in the guideline that we're quoting (some text is bold), I was trying to avoid stylizing the quotes. I'll try a couple options to make the blockquotes stand out a bit.

Reply:
the color approach looks good to me